test/automated_serial_generation_e2e_test.ts

```
@@ -645,39 +645,52 @@
 
-    // Start Electron app with electron-forge directly to avoid PATH and shell security warnings
-    // Resolve cross-platform executable (Windows needs .cmd extension)
-    const electronForgeBinary = process.platform === "win32" ? "electron-forge.cmd" : "electron-forge";
-    const electronForgeBinPath = path.join(process.cwd(), "node_modules", ".bin", electronForgeBinary);
-
-    resources.electronProcess = spawn(
-      process.execPath, // Node.js executable path
-      [electronForgeBinPath, "start"], // Arguments: [script path, "start"]
-      {
+    const shouldSpawnElectron =
+      process.env.SKIP_ELECTRON_SPAWN !== "1" && process.env.SKIP_ELECTRON_SPAWN !== "true";
```

ESLint

[error] Delete `⏎·····`

(prettier/prettier)